### ISSUE TITLE
Print Meal - recipe

### ISSUE THREAD

**Issue by shanthisa** [CivicTechFredericton/mealplanner#369]
Print meal recipe - In print view, the meal recipe should not have the youtube video embedded, but display the method, ingredients, etc, meal description, attributes, tags, price, nutrition rating

**Comment by BawbSmith** [comment]
In addition, we don't need Users comments (description) or the measurement.

**Comment by shanthisa** [comment]
I am not sure what do you mean by users comments. There is no comments section in a meal. The description of meal is required. Also measurement (Measurement is the ingredients) is required.

**Comment by BawbSmith** [comment]
I would suggest that Description be used for user to write comments about why she created this plan and follow-up comments about plan success. Use Tags to identify actual (abbreviated) meal name - much more informative. Example use 'chili' instead of 'spicy'. How do you describe a meal plan with possible 28  mealtimes with multiple meal courses. User would write comments.

**Comment by BawbSmith** [comment]
Also...I didn't mention, my comment on description and measure was in reference to the Shopping list, not needing this info.

**Comment by shanthisa** [comment]
Your comments are not relevant to this ticket. This ticket is created for the feature request of Print meal in Meal view.

**Comment by BawbSmith** [comment]
Very sorry. I don't know my way around github, who to talk to about what. it's not my place to join so late and suggest anything.  I  tried deleting meals of comments in admin but couldn't. I will delete my meal plans of comments too. I'll back off for a while. I have done a thorough QA, tho I wasn't asked to, I have spent much time reviewing and yes critiquing the entire application and I have notes. But shanthisa, I never at any time meant to criticize your efforts. I spent hours, you have spent years putting tremendous effort into a product I know GV will love. I meant no offence by any comment.
